src/SPC/command/BuildPHPCommand.php

```diff
@@ -107,18 +126,22 @@ public function handle(): int
             $builder = BuilderProvider::makeBuilderByInput($this->input);
             $include_suggest_ext = $this->getOption('with-suggested-exts');
             $include_suggest_lib = $this->getOption('with-suggested-libs');
-            [$extensions, $libraries, $not_included] = DependencyUtil::getExtsAndLibs($extensions, $libraries, $include_suggest_ext, $include_suggest_lib);
+            [$extensions, $libraries, $not_included] = DependencyUtil::getExtsAndLibs(array_merge($static_extensions, $shared_extensions), $libraries, $include_suggest_ext, $include_suggest_lib);
```

Comment:
I think here is not approperiate. We need to use `$extensions` as static extensions `$static_extensions`. (or separate? I don't know)

Reply:
Hmm, I'm not sure why this wouldn't be fine. My thought here is that we need to compile all required extensions dependencies, no matter if shared or static.

Reply:
I've added a filter for `$static_extensions` to solve this problem.